src/main/resources/public/src/views/Dashboard.vue

```diff
@@ -1,13 +1,54 @@
 <template>
-    
+    <div id="container" class="container">
+        <h2>Dashboard</h2>
+        <table class="table table-sm">
+            <thead class="thead-dark">
+                <tr>
+                    <th>Nom</th>
+                    <th>DC Model</th>
+                    <th>Organization</th>
+                    <th>State</th>
+                </tr>
```

Comment:
I think that it would be better to not have this on dashboard but only a link to the page containing this board.
Plus, I think the board should have a date (meaning adding it in mongoDB model) for each form, and only take the organization siret instead of all the URI